packages/kolibri/composables/__mocks__/useUser.js

```diff
@@ -77,7 +78,29 @@ export function useUserMock(overrides = {}) {
   for (const key in mocks) {
     computedMocks[key] = computed(() => mocks[key]);
   }
-  return computedMocks;
+
+  // Module-level state reference for actions
+  const session = ref({ ...mocks.session });
+
+  // Mock implementation of `useUser` methods
+  return {
+    ...computedMocks,
+    session, // Make session mutable for test scenarios
+
+    // Actions
+    setSession: jest.fn(({ session: newSession, clientNow }) => {
```

Comment:
One of the failing tests re: not calling `serverTime` w/ `clientNow` may be fixed by calling it here.